contracts/test/oracle/oracle.fork-test.js

```diff
@@ -0,0 +1,58 @@
+const { expect } = require("chai");
+const { parseUnits } = require("ethers/lib/utils");
+
+const { loadDefaultFixture } = require("../_fixture");
+const { forkOnlyDescribe, isCI } = require("../helpers");
+
+forkOnlyDescribe("ForkTest: Oracle Routers", function () {
+  this.timeout(0);
+
+  // Retry up to 3 times on CI
+  this.retries(isCI ? 3 : 0);
+
+  let fixture;
+
+  describe("OETH Oracle Router", () => {
+    let oethOracleRouter;
+    beforeEach(async () => {
+      fixture = await loadDefaultFixture();
+      oethOracleRouter = await ethers.getContract("OETHOracleRouter");
```

Comment:
Are there any downsides from loading this from the fixture object? (That's at least how we've handled getting contracts from fixtures in the past, I think.)